inspire_mitmproxy/services/base_service.py

```diff
@@ -105,6 +110,12 @@ def increment_interaction_count(self, interaction_name: str):
                 {'num_calls': 1},
             )
 
+    def get_interaction_count(self, interaction_name: str) -> int:
```

Comment:
(I am so sorry for this :slightly_smiling_face:) Maybe `get_interaction_replay_count` instead? Otherwise could be confused for the number of interactions themselves...